arm-logic/2016-06-01/examples/AgreementsCreateOrUpdate.json

```diff
@@ -0,0 +1,485 @@
+{
+   "parameters":{
+      "api-version":"2016-06-01",
+      "resourceGroupName":"<resourceGroupName>",
```

Comment:
please provide sample values everywhere. "<resourceGroupName>" does not match the regex of resourceGroupName `[ '^[-\\w\\._\\(\\)]+$'`